src/Build/BackEnd/Components/FileAccesses/FileAccessManager.cs

```diff
@@ -107,6 +134,25 @@ private void UnregisterHandlers(Action<BuildRequest, FileAccessData> fileAccessH
             }
         }
 
+        public static void NotifyFileAccessCompletion(int globalRequestId)
+        {
+            // Make a dummy file access to use as a notification that the file accesses should be completed for a project.
+            string filePath = FileAccessCompletionPrefix + globalRequestId.ToString();
+            _ = File.Exists(filePath);
```

Comment:
Does this not throw with characters like `{`?

Reply:
`{` isn't invalid :)

![image](https://github.com/dotnet/msbuild/assets/6445614/303dd060-316f-47ee-beea-5cbe9bc2ec22)

Reply:
😮